spec/lib/landable/bunny_messaging_service.rb

```diff
@@ -0,0 +1,8 @@
+class BunnyMessagingService
+  def initialize
```

Comment:
where do you want this documentation, is what is in the generator sufficient?  if not, where should I put the docs?